HTMLtask1/templ1.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>Ellipsis</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
+  <link rel="stylesheet" type="text/css" media="screen" href="templStyle1.css" />
+</head>
+<body>
+  <header class="main-page">
+      <div class="main">
+        <img src="pics/mainLabel.png" alt="Triangle label" id="triangleM">
+        <div class="main-head">Working<br> with design lab</div>
+        <div class="main-par">We’ve been helping startups launch and grow their businesses.<br>
```

Comment:
what is `main-par`? If it's paragraph, you can use `p` instead of `div`. It's more efficient

Reply:
but mentor suggested to use divs

Reply:
as you wish

Reply:
i changed for more semantic elements, it's completed now, but i have problems with responsiveness